src/pages/lock-exception/LockException.js

```diff
@@ -520,18 +524,16 @@ class LockException extends Page {
                                 pageState: newPageState,
                             })
                         } else {
-                            this.props.deleteInProgress = true
-                            this.context.updateAppState({
-                                pageState: newPageState,
-                            })
```

Comment:
I've removed this part, because after deletion it should just fetch a fresh list from the server.